cpp/tests/stream_compaction/drop_duplicates_tests.cpp

```diff
@@ -14,254 +14,245 @@
  * limitations under the License.
  */
 
-#include <algorithm>
-#include <cmath>
-#include <ctgmath>
 #include <cudf/copying.hpp>
+#include <cudf/sorting.hpp>
 #include <cudf/stream_compaction.hpp>
 #include <cudf/table/table.hpp>
 #include <cudf/table/table_view.hpp>
 #include <cudf/types.hpp>
+
 #include <cudf_test/base_fixture.hpp>
 #include <cudf_test/column_utilities.hpp>
 #include <cudf_test/column_wrapper.hpp>
 #include <cudf_test/table_utilities.hpp>
 #include <cudf_test/type_lists.hpp>
 
+#include <algorithm>
+#include <cmath>
+#include <ctgmath>
```

Comment:
According to https://en.cppreference.com/w/cpp/header/ctgmath, this header is deprecated in C++17 and removed in C++20. Include `<cmath>` or `<complex>` as appropriate, instead.

We don't deal with complex types in cuDF so it's probably just the same as including `<cmath>`, which is already present -- so just remove this line?
```suggestion
```